README.md

```diff
@@ -24,8 +24,6 @@ Tasks to do:
 
```

Comment:
- Add instructions on how to install and run your project locally. Keep your README file professional, like your project. It's the first you when somebody watch on your repository.